Frontend/src/data/rental/inputs.js

```diff
@@ -346,6 +373,10 @@ export default {
       component: AutocompleteInput,
       nobind: true,
       props: {
+        localSorting: true,
+        sortByMatchedKeywords: true,
+        itemSortFunction: () => sortItemByIdOrName,
```

Comment:
```suggestion
        itemSortFunction: sortItemByIdOrName,
```

Reply:
nope, needs to stay like this, else it doesnt work... no idea why!